department-of-veterans-affairs/caseflow
# APPEALS-23521 CAVC Dashboard not showing all issues for Remand

Resolves [23521](https://jira.devops.va.gov/browse/APPEALS-23521)

# Description
- Changed CAVC Dashboard to show all issues from source appeal
- Fix CAVC Dashboard not created for some remand types

## Acceptance Criteria

## Testing Plan

For 23520
1. Login as CAVC Litigation Admin
1. On the "Your cases" page, click the "Search cases" link to be taken to the "Veteran Case Search" page
2. On the "Veteran Case Search" page, enter a Veteran ID to locate an existing Appeal (4 million series)
3. On the results page, click on the docket number of a case to add a CAVC Remand to
4. Click the "Add CAVC Remand" button 
5. Choose one of the following "How are you proceeding" options:
     - Affirmed
     - Dismissal, other than death
     - Settlement
6. Fill out remaining info and click the "Submit" button
7. Login as an OCC and/or OAI team member.
8. On the "Your cases" page, click the "Search cases" link to be taken to the "Veteran Case Search" page
9. Search the veteran ID you used to create a CAVC Remand
10. Go into the source appeal case details page
11. Click on the CAVC Dashboard button
12. Observe that a CAVC Dashboard is created for the new remand you just made.

For 23521
1. Login as CAVC Litigation Admin
1. On the "Your cases" page, click the "Search cases" link to be taken to the "Veteran Case Search" page
2. On the "Veteran Case Search" page, enter a Veteran ID to locate an existing Appeal (4 million series)
3. On the results page, click on the docket number of a case to add a CAVC Remand to
4. Click the "Add CAVC Remand" button 
5. Choose one of the following "How are you proceeding" options:
     - Remand
     - Straight Reversal
6. On the "Which issues are being addressed by the court?" section of the form, do not check all the issues, only have some checked.
7. Fill out remaining info and click the "Submit" button
8. Login as an OCC and/or OAI team member.
9. On the "Your cases" page, click the "Search cases" link to be taken to the "Veteran Case Search" page
10. Search the veteran ID you used to create a CAVC Remand
11. Go into the source appeal case details page
12. Click on the CAVC Dashboard button
13. Click on the CAVC Dashboard tab that you just created.
14. Observe that all issues are present and not just the CAVC Appeal issue

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
